bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/target/NonSiderealTargetRules.scala

```diff
@@ -0,0 +1,86 @@
+package edu.gemini.p2checker.target
+
+import edu.gemini.p2checker.api.{P2Problems, IP2Problems, ObservationElements, IRule}
+import edu.gemini.shared.util.immutable.ScalaConverters._
+import edu.gemini.spModel.core._
+import scalaz._, Scalaz._
+
+final class NonSiderealTargetRules extends IRule {
+  import NonSiderealTargetRules._
+
+  implicit class SemesterOps(s: Semester) {
+    def contains(time: Long, site: Site): Boolean =
+      time >= s.getStartDate(site).getTime &&
+      time <=   s.getEndDate(site).getTime
+  }
```

Comment:
Seems like maybe something that is useful enough to go into `Semester` itself.  It's surprising we've never needed it before.